core/src/services/cos/backend.rs

```diff
@@ -428,6 +429,7 @@ impl Access for CosBackend {
                 self.core
                     .cos_put_object_request(path, None, v, Buffer::new())?
             }
+            PresignOperation::Delete(v) => todo!(),
```

Comment:
idk how to implement all these services.

Reply:
No need; we can implement them gradually and across multiple PRs.